deep_conmech/simulator/mesh/mesh_builders_2d.py

```diff
@@ -1,4 +1,5 @@
 from ctypes import ArgumentError
+
 # import dmsh
```

Comment:
Can you optimize and clean imports (here and in other places)?

Reply:
> And can you add example (e.g. which generate results which you submitted to me today)? I'm thinking about one short file with all parameters which can serve as a demo.

Done - placed it in "examples"